src/index.js

```diff
@@ -387,6 +412,39 @@ function fixTouchableMixin(t, path) {
   });
 }
 
+function extendReactElementWithRef(path) {
+  if (path.node.key.name === 'ref' && t.isIdentifier(path.node.value)) {
```

Comment:
Ohhh, I get it, ok.  So this touches every instance of `ref:` in *every* objectproperty, huh, because we can't look up a function named `ReactElement` because it got minified.  Uh, hmm.  Could be nice to make sure that the `$$typeof` is going to be `react.element` -- actually, that would be even *more* bulletproof than just looking for the function `ReactElement`!  That would mean that we'd have to find a sibling of the `path`, make sure that the sibling `$$typeof` is there, look up the variable, and look that up in the `state` maybe?  Let me mess with this a little if I can figure out how to run Babel.  You've got my curiosity here.